src/features/collection/constants/index.ts

```diff
@@ -1,3 +1,42 @@
+import { StaticImageData } from 'next/image';
+
+import logical from '../../../../public/images/capabilites/logical.png';
+import performance from '../../../../public/images/capabilites/performance.png';
+import relationship from '../../../../public/images/capabilites/relationship.png';
+import selfBased from '../../../../public/images/capabilites/self_based.png';
+
+export const ANSWER_MAX_LENGTH = 2000;
+
+type CapabilitiesImage = {
+  [key: string]: StaticImageData;
+};
```

Comment:
```suggestion
type CapabilitiesImage = Record<string, StaticImageData>;
```
Record 타입으로 변환 할 수도 있겠네요!